service/src/test/java/tds/assessment/web/endpoints/AssessmentControllerTest.java

```diff
@@ -10,50 +10,51 @@
 import org.springframework.web.context.request.ServletRequestAttributes;
 
 import javax.servlet.http.HttpServletRequest;
+import java.util.ArrayList;
 import java.util.Optional;
 
-import tds.assessment.SetOfAdminSubject;
-import tds.assessment.services.AdminSubjectService;
+import tds.assessment.Assessment;
+import tds.assessment.services.AssessmentService;
 import tds.common.web.exceptions.NotFoundException;
 
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
 
-public class AdminSubjectControllerTest {
-    private AdminSubjectController controller;
-    private AdminSubjectService service;
+public class AssessmentControllerTest {
+    private AssessmentController controller;
+    private AssessmentService service;
 
     @Before
     public void setUp() {
         HttpServletRequest request = new MockHttpServletRequest();
         ServletRequestAttributes requestAttributes = new ServletRequestAttributes(request);
         RequestContextHolder.setRequestAttributes(requestAttributes);
 
-        service = mock(AdminSubjectService.class);
-        controller = new AdminSubjectController(service);
+        service = mock(AssessmentService.class);
```

Comment:
Mind adding @RunWith(MockitoJunitRunner.class) and using the @Mock mockito annotation above the service?  @jeffjohnson9046 rolls with that style and I agreed to follow it.  This just hasn't been converted.